utils/yoke/derive/Cargo.toml

```diff
@@ -27,4 +27,4 @@ synstructure = "0.12.4"
 
 [dev-dependencies]
```

Comment:
note: when publishing yoke-derive you _may_ need to blank the dev-dependencies section and use --allow-dirty